etc/sql/oracle/procedures.sql

```diff
@@ -521,8 +526,28 @@ FOR i IN ( SELECT scope FROM scopes WHERE scope NOT LIKE 'mock%' AND scope NOT I
     TIER,
     SPACETOKEN,
     GRP_DATATYPE,
-    SITE)
-        SELECT /*+ LEADING(D R RS) USE_HASH(RS) USE_HASH(R) SWAP_JOIN_INPUTS(RS) INDEX_FFS(RS ("RSES"."ID")) FULL(R) FULL(D) USE_HASH_AGGREGATION */ 
+    SITE,
+    REP_TYPE)
+        WITH l AS
+         (SELECT /*+ use_hash(a,b) full(a) full(b) */ a.scope,
+                                                      a.name,
+                                                      a.rse_id,
+                                                      MAX(a.bytes) AS fbytes,
+                                                      MAX(CASE
+                                                              WHEN b.locked=1 THEN 2

```

Comment:
shouldn't there be a fourth type for the caches? :thinking:

Reply:
See `NVL(l.rep_type, 4)` line 577
4 = cached
2 = custodial
1 = permanent
0 = temporary